pkg/controllers/disruption/helpers.go

```diff
@@ -85,18 +85,18 @@ func simulateScheduling(ctx context.Context, kubeClient client.Client, cluster *
 	if _, ok := lo.Find(deletingNodes, func(n *state.StateNode) bool {
 		return candidateNames.Has(n.Name())
 	}); ok {
-		return nil, errCandidateDeleting
+		return pscheduling.Results{}, errCandidateDeleting
```

Comment:
nit: I think it's kind of awkward that we call this package `pscheduling` which is supposed to mean "provisioner scheduling" but we are using it in the context of disruption. I know that we do this elsewhere but I wonder if we just call this the "scheduling" package and the other thing is scheduling utils or something similar

Reply:
I'd rather just leave as is, since we follow the same naming scheme elsewhere. Could consider moving it, but also I like how the other reads `scheduling.Requirements` whereas this one is just `pscheduling.Results`, which I think is fine.